tools/msi/product.wxs

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
-  <?define ProductName="NexPloit CLI" ?>
+  <?define ProductName="Bright CLI" ?>

```

Comment:
To ensure that installations are not performed out of order and to properly update the `Id` value, simply changing names in Wix files is insufficient. It is necessary to establish an upgrade policy.